modules/misc/news.nix

```diff
@@ -2029,6 +2029,25 @@ in
           A new module is available: 'programs.foot'.
         '';
       }
+
+      {
+        time = "2021-05-19T01:26:16+00:00";
+        message = ''
+          The xmonad module now compiles the configuration beforing
+          linking the binary to the place xmonad expects to find
+          the compiled configuration (the binary).
```

Comment:
Maybe I don't understand xmoad, but I find this description to be quite confusing.

I think my biggest confusion is what is the binary? Is it the config? Is it the xmonad executable? Maybe splitting this into two sentences would help.

Reply:
`xmonad` inherits from the [suckless](https://suckless.org/) philosophy the trait that its configuration is part of the program, therefore written in the program's implementation language (here: Haskell), and *compiled into the binary*.  In that sense, the sentence is semantically correct, but a bit hard to untangle.